code/src/components/ScanResultPage.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import { useSelector } from 'react-redux'
+import './scanresultpage.css'
+
+export const ScanResultPage = () => {
+  const productInfo = useSelector((state) => state.products.productinfo)
+
+  // !productInfo = vi har inte hämtat något ännu
+  if (!productInfo) {
+    return null
+  }
+
+  if (productInfo.status !== 1) {
+    return <h1>Oh no! Product not found!</h1>
+  }
```

Comment:
Super! Maybe also fix so it's only possible to write numbers? If you write letters now, it doesn't get "caught" in this safety check.